projects/schema-form/src/lib/model/formproperty.ts

```diff
@@ -233,7 +233,7 @@ export abstract class FormProperty {
     expression: string|string[] = ''): boolean {
```

Comment:
If we want to support number here, the type must be `string|string[]|number`.
(and as a matter of fact, it is breaking the tests: https://travis-ci.org/github/guillotinaweb/ngx-schema-form/builds/665277673)